src/index.js

```diff
@@ -554,7 +554,10 @@ export default (Bookshelf, options = {}) => {
 
                 _forEach(sortValues, (sortBy) => {
 
-                    internals.model.orderBy(sortBy, sortDesc.indexOf(sortBy) === -1 ? 'asc' : 'desc');
+                    internals.model.orderBy(
+                        sortBy && internals.formatRelation(sortBy),
```

Comment:
Not quite sure about this line, why not only pass in internals.formatRelation(sortBy)?

Reply:
In the case where `sortBy` is an empty string or null/undefined, `formatRelation(sortBy)` returns `modelName.` which makes the query fail.

Reply:
If that is the case, then I think we should put a conditional around the orderBy:
```
if (sortBy){
internals.model.orderBy(...)
}
```